### PR TITLE
Robin fix FAQs

### PR DESCRIPTION
**Pull request by robin-norwood**
This one probably needs a careful look.

### CODE REVIEW

File: content/cloud-block-storage/cloud-block-storage.md [hunk 1]
```diff
@@ -42,4 +42,4 @@ product_url: cloud-block-storage
 ###  API Developer Guide & FAQ's
```
Comment (kmsholcomb) [inline content/cloud-block-storage/cloud-block-storage.md]: Also here change FAQ's to FAQ

File: content/cloud-sites/important-scheduled-maintenance-dfw-environment-migration.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ Center, which includes FAQ&rsquo;s for individual technologies. Updated
 testlink information will be provided as a courtesy to review your site
 functionality prior to the migration.
 
-Please review the links below which include FAQs & helpful information
+Please review the links below which include FAQ & helpful information
 on common break changes for your specific technology during to prepare
```
Comment (kmsholcomb) [inline content/cloud-sites/important-scheduled-maintenance-dfw-environment-migration.md]: Change instead to "Please review the following links, which provide helpful information ..."

File: content/managed-operations/managed-operations-faq.md [hunk 3]
```diff
@@ -155,7 +155,7 @@ infrastructure layer support, and technical guidance for your Cloud
 Servers.
 
 -   [Rackspace Cloud Backup:
```
Comment (kmsholcomb) [inline content/managed-operations/managed-operations-faq.md]: Remove the colon after Backup.

File: content/rackspace-email/rackspace-email-faq.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 node_id: 4977
-title: 'Rackspace Email: FAQs'
+title: 'Rackspace Email: FAQ'
```
Comment (kmsholcomb) [inline content/rackspace-email/rackspace-email-faq.md]: Remove the colon preceding FAQ

File: content/cloud-servers/general-purpose-cloud-servers-how-to-articles-and-other-resources.md [hunk 5]
```diff
@@ -46,7 +46,7 @@ articles:</span></span></span>
 
 <span><span><span>&middot;</span><span>
 </span></span></span>[<span><span><span>Cloud Servers:
```
Comment (kmsholcomb) [inline content/cloud-servers/general-purpose-cloud-servers-how-to-articles-and-other-resources.md]: Remove the colon after Servers

File: content/cloud-servers/cloud-servers-how-to-articles-other-resources.md [hunk 6]
```diff
@@ -44,7 +44,7 @@ articles:</span></span></span>
 
 <span><span><span>&middot;</span><span>
 </span></span></span>[<span><span><span>Cloud Servers:
-FAQs</span></span></span>](/how-to/product-faq/cloud-servers)
```
Comment (kmsholcomb) [inline content/cloud-servers/cloud-servers-how-to-articles-other-resources.md]: Remove the colon after Servers

**Comment by kmsholcomb** [comment]
@robin-norwood: I made some comments. Do you want to make the changes, or can I?

**Comment by robin-norwood** [comment]
You go for it if you don't mind - I'm going to be in meetings for most of the rest of the day.

**Comment by kmsholcomb** [comment]
OK. I'll give it a shot.

**Pull request merged**